books/protocol/src/consensus_rules/blocks.md

```diff
@@ -17,14 +18,16 @@ are different to normal transactions.
 
 ### Block Weight And Size
 
-The `block blob` must not be bigger than (2 * the [effective median weight](./blocks/weights.md#effective-median-weight) + 100)[^block-size-check].
+The `block blob` must not be bigger than (2 *
+the [effective median weight](./blocks/weights.md#effective-median-weight) + 100)[^block-size-check].
 
```

Comment:
Your editor sometimes formats sentences a bit awkwardly btw.

Reply:
hmm strange I'll try turn that off.